abi/sector.go

```diff
@@ -0,0 +1,164 @@
+package abi
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-state-types/big"
+)
+
+// SectorNumber is a numeric identifier for a sector. It is usually relative to a miner.
+type SectorNumber uint64
+
+func (s SectorNumber) String() string {
+	return strconv.FormatUint(uint64(s), 10)
+}
+
+// The maximum assignable sector number.
+// Raising this would require modifying our AMT implementation.
+//goland:noinspection GoUnusedConst
```

Comment:
Are these here on purpose?  I'd rather keep editor specific directives out of the source.  Planning to remove but we can revert later today.